src/main/java/uk/gov/companieshouse/chdorderconsumer/service/ItemOrderedProcessorService.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.companieshouse.chdorderconsumer.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.model.order.chd.MissingImageDeliveryRequestApi;
+import uk.gov.companieshouse.chdorderconsumer.exception.RetryableErrorException;
+import uk.gov.companieshouse.chdorderconsumer.exception.ServiceException;
+import uk.gov.companieshouse.orders.items.ChdItemOrdered;
+import uk.gov.companieshouse.orders.items.Item;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.CREATED;
+import static org.springframework.http.HttpStatus.UNAUTHORIZED;
+
+@Service
+public class ItemOrderedProcessorService {
+
+    private static final String POST_MISSING_IMAGE_CHD_ORDER_URI = "/chd-order-api/missing-image-deliveries";
+
+    private final CHDOrderService chdOrderService;
+
+    public ItemOrderedProcessorService(final CHDOrderService chdOrderService) {
+        this.chdOrderService = chdOrderService;
+    }
+
+    public MissingImageDeliveryRequestApi mapChdItemOrderedToMissingImageDeliveryRequestApi(ChdItemOrdered chdItemOrdered) {
```

Comment:
Does this need to be public?

Reply:
Yes, but for the unit test.

Reply:
Could use package (default) access level for that.

Reply:
I have changed this